build-tools/packages/build-tools/package.json

```diff
@@ -53,6 +53,7 @@
 		"find-up": "^7.0.0",
 		"fs-extra": "^11.2.0",
 		"glob": "^7.2.3",
+		"globby": "^11.1.0",
```

Comment:
globby is very helpful because it automatically handles .gitignore, which is a pain to do with glob.

Ideally I'd replace all the "glob" usages to globby and just have one dep, but I think that can be done separately.

Reply:
Ultimately used didn't use globby anyway